### ISSUE TITLE
[Enhancement] External lightweight application for Mac and Bedrock users

### ISSUE THREAD

**Issue by 42willow** [plasmoapp/plasmo-voice#353]
Mac users need to use Prism launcher or something else at the moment to work, and bedrock users joining via GeyserMC can't use this at all. SimpleVoiceChat has the same issue. A lightweight application with a little GUI would be perfect for solving this issue.

This would put plasmo above SVC for sure ;)

But.. this may be pointless because I just realised https://openaudiomc.net exists, but without a self-hosting option, and is less integrated.

tl;dr
An external app or web interface which has a simple GUI for proximity chat. Useful for users without access to the mod.
